_posts/2023-06-20-trino-fest-2023-recap.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Trino Fest 2023 recap"
+author: "Manfred Moser, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/trino-fest.png
+---
+
+Last week we held Trino Fest, and we did not find time to chill at the
+lakehouse. Despite the theme of a summercamp for lakehouse analytics, the fest
+kept us all busy. Great demos, amazing announcements, new plugins, and use cases
```

Comment:
I feel like this is a bit of a negative tone for starting things off. Maybe re-order to, "and it kept us all so busy, we forgot to spend time chilling by the lakehouse!" That feels a little more cheeky and fun IMO.

Reply:
Done